app/scripts/controllers/detect-tokens.js

```diff
@@ -43,7 +45,10 @@ export default class DetectTokensController {
     tokensController,
     assetsContractController = null,
     trackMetaMetricsEvent,
+    getNetworkClientById,
```

Comment:
this is required now, and is a breaking change.

need to change the instance of `new DetectTokensController` in the code to add `getNetworkClientById` to the constructor options

Reply:
Ok looks like it might be broken because you're not passing this in `metamask-controller`...?